content/influxdb/v2.2/reference/api/influxdb-1x/dbrp.md

```diff
@@ -63,12 +67,19 @@ the DBRP mapping service checks for a bucket mapped to the database and retentio
 
 ### When querying data
```

Comment:
```suggestion
### Query mapped buckets
```

Reply:
With this being a reference doc that describes the behaviors of DBRP maps, imperative voice doesn't feel right here.